BHoM_UI/Templates/MethodCaller.cs

```diff
@@ -133,58 +112,14 @@ public override object Run(object[] inputs)
             {
                 BH.Engine.Reflection.Compute.RecordError("The component is not linked to a method.");
                 return null;
-            }    
+            }
         }
 
 
         /*************************************/
         /**** Private Methods             ****/
         /*************************************/
 
-        protected virtual void CompileFunction()
-        {
-            if (Method == null)
-                return;
-
-            ParameterExpression lambdaInput = Expression.Parameter(typeof(object[]), "x");
-            Expression[] inputs = Method.GetParameters().Select((x, i) => Expression.Convert(Expression.ArrayIndex(lambdaInput, Expression.Constant(i)), x.ParameterType)).ToArray();
-
-            if (Method is MethodInfo)
-            {
-                MethodCallExpression methodExpression = Expression.Call(Method as MethodInfo, inputs);
-                m_CompiledFunc = Expression.Lambda<Func<object[], object>>(Expression.Convert(methodExpression, typeof(object)), lambdaInput).Compile();
-            }
-            else if (Method is ConstructorInfo)
-            {
-                NewExpression constructorExpression = Expression.New(Method as ConstructorInfo, inputs);
-                m_CompiledFunc = Expression.Lambda<Func<object[], object>>(Expression.Convert(constructorExpression, typeof(object)), lambdaInput).Compile();
-            }
-            
-        }
-
-        /*************************************/
-
-        protected Type GetConstructedType(Type type)
```

Comment:
This has now been migrated to the Reflection_Engine as `Engine.Reflection.Modify.MakeGeneric`